Dockerfile.template

```diff
@@ -1,2 +1,4 @@
 LABEL io.openshift.expose-services="3306:tcp" \
     io.openshift.tags="component,%NAME%"
+RUN apt-get update && apt-get install -y gettext-base python &&\
+    cat /opt/app-root/etc/my.cnf > /etc/mysql/my.cnf
```

Comment:
I think it'll be better to use the `MYSQL_HOME` variable https://mariadb.com/kb/en/mariadb/configuring-mariadb-with-mycnf/